### PR TITLE
Add prod approval list

### PR DESCRIPTION
**Pull request by njrich28**

### CODE REVIEW

File: environment-approvals.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+prod:
+  - https://github.com/HMCTS/cnp-plum-recipes-service.git
```
Comment (timja) [inline environment-approvals.yml]: I would remove hmcts and .git 👅 
should be easy to remove?
```suggestion
  - cnp-plum-recipes-service
```

File: environment-approvals.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+  - https://github.com/HMCTS/cnp-plum-shared-infrastructure.git
```
Comment (timja) [inline environment-approvals.yml]: I also wonder whether we should have a yaml object to make it more extensible, 
```suggestion
  - repo: https://github.com/HMCTS/cnp-plum-shared-infrastructure.git
```

File: environment-approvals.yml (hunk 1 above)
Comment (njrich28) [reply environment-approvals.yml]: i was thinking about future where we might be using something else e.g. azure DevOps

Comment (timja) [reply environment-approvals.yml]: we can do the same there, and I don't think there would be any plan to move code to azure devops? we can't code in the open there...

File: environment-approvals.yml (hunk 2 above)
Comment (njrich28) [reply environment-approvals.yml]: was thinking similar - but was wondering if it's too much thinking ahead

File: environment-approvals.yml (hunk 1 above)
Comment (njrich28) [reply environment-approvals.yml]: would prefer to keep it explicit - it's a simpler comparison and avoids any potential bugs/edgecases in stripping this out or hard-coding prefixes and suffixes, etc

Comment (timja) [inline environment-approvals.yml]:
```suggestion
  - repo: https://github.com/HMCTS/cnp-plum-recipes-service.git
```

**Pull request merged**